src/main/java/address/shortcuts/ShortcutsManager.java

```diff
@@ -41,6 +49,9 @@
 
         SHORTCUT_FILE_SAVE_AS = setShortcut(KeyCode.S, KeyCombination.CONTROL_DOWN, KeyCombination.ALT_DOWN);
 
+        HOTKEY_APP_MINIMIZE = setHotkey("control alt X", new MinimizeAppRequestEvent());
+        HOTKEY_APP_MAXIMIZE = setHotkey("control shift X", new MaximizeAppRequestEvent());
```

Comment:
For Mac, `command` button is preferred as shortcut button instead of `control`. That can be achieved by using `"meta alt X"` for minimize case (tested).